manage.py

```diff
@@ -0,0 +1,213 @@
+            archive_tables_dir = '%s/abdump' % archive_dir
+            create_path(archive_tables_dir)
+            cursor = db_connection.cursor()
+            for table in _tables.keys():
+                print(" - Dumping %s table..." % table)
+                with open('%s/%s' % (archive_tables_dir, table), 'w') as f:
+                    cursor.copy_to(f, table, columns=_tables[table])
+            tar.add(archive_tables_dir, arcname='abdump')
+
+            shutil.rmtree(temp_dir)  # Cleanup
+
+        pxz.stdin.close()
+
+    print("Database dump created:\n +", archive_path)
+
+    if rotate:
+        print("Removing old dumps (except two latest)...")
+        remove_old_archives(location, "acousticbrainzdump-[0-9]+-[0-9]+.tar.xz",
+                            is_dir=False, sort_key=lambda x: os.path.getmtime(x))
+
+    print("Done!")
+
+
+@manager.option('-nh', '--no-highlevel', dest='no_highlevel', action='store_true',
+                help="Don't dump high level data.", default=False)
+@manager.option('-nl', '--no-lowlevel', dest='no_lowlevel', action='store_true',
+                help="Don't tump low level data.", default=False)
+def export_json(no_highlevel, no_lowlevel):
+    if no_highlevel and no_lowlevel:
+        print("wut? check your options, mate!")
+
+    if not no_highlevel:
+        export_highlevel_json()
+
+    if not no_lowlevel:
+        export_lowlevel_json()
+
+
+@manager.command
+def export_lowlevel_json(location=os.path.join(os.getcwd(), 'export'), rotate=False):
+    print("Dumping lowlevel data (JSON)...")
+    print("Done! Created:", dump_lowlevel_json(location))
+
+    if rotate:
+        print("Removing old sets of archives (except two latest)...")
+        remove_old_archives(location, "acousticbrainz-lowlevel-json-[0-9]+-json.tar.bz2",
+                            is_dir=False, sort_key=lambda x: os.path.getmtime(x))
+
+
+@manager.command
+def export_highlevel_json(location=os.path.join(os.getcwd(), 'export'), rotate=False):
+    print("Dumping highlevel data (JSON)...")
+    print("Done! Created:", dump_highlevel_json(location))
+
+    if rotate:
+        print("Removing old sets of archives (except two latest)...")
+        remove_old_archives(location, "acousticbrainz-highlevel-json-[0-9]+-json.tar.bz2",
+                            is_dir=False, sort_key=lambda x: os.path.getmtime(x))
+
+
+def _run_sql_script(sql_file):
+    cursor = db_connection.cursor()
+    with open(sql_file) as sql:
+        cursor.execute(sql.read())
+
+
+def _import_data(archive):
+    """Import data from .tar.xz archive into the database."""
+    pxz_command = ['pxz', '--decompress', '--stdout', archive]
+    pxz = subprocess.Popen(pxz_command, stdout=subprocess.PIPE)
+
+    table_names = _tables.keys()
+    cursor = db_connection.cursor()
+
+    with tarfile.open(fileobj=pxz.stdout, mode='r|') as tar:
+        for member in tar:
+
+            if member.name == 'SCHEMA_SEQUENCE':
```

Comment:
*SCHEMA_SEQUENCE* is added to tar file first during creation, so it should be a first member there. We can check if it is.